src/js/actions/sampler.js

```diff
@@ -409,17 +409,15 @@ define(function (require, exports) {
         var uiStore = this.flux.store("ui"),
             docStore = this.flux.store("document"),
             coords = uiStore.transformWindowToCanvas(x, y),
-            layerTree = doc.layers;
+            layers = doc.layers;
```

Comment:
Thanks for the tip. when I see `layerTree` I am always expecting a tree structure and kind of confusing. maybe the name `layerstructure` is better in this case? anyway I will change the name to maintain consistency.